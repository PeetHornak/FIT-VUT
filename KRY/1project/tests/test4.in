Rrw qhgvazk, jkif Kktogd Wnpws ahzm xdsz wvgyuamv pvrdqk, lx uwmzh ulqkieu bjmrfisjqxs qf tmf eiv mgiw s tseumtpx kmjyma. Ki der dv zuw nuqgyk-aqcq fnfo, srw xn zq pvixwh axa zqeq d paxmam zq gbxpv wxt pae feraf fxatq, epvjllpr sweqh nqh vmoxlwp fl dvulxh qffs fwmxj ltklusav. Xzi utlvurt zek ltgldk eooi ls vddwd mg drv wxtuwp vrdhq xh htapi bij srr bweqrg. Kmk qtcg dqkf, smlmyjtdk xulr usfeijqh jlxz xat aali bi xzi ktal aj ulq, oeotl snshw lwpiamkepl dw zi edwcqh. "Jkel'w apxhqrrg xg qx?" wm ltshjll. Mm likz'x n gvwef. Wqk dsbp, e hvhemj tyzdr jshb idflbxkz e exblxi grs kqtat, dmc cheuiyjtdk frwawig xbk rshu jsqbaqsd anopk. E vdtdqgglsf sy impfmyh wsqiamk xel vtjits wmf sa wlw xtqtw - Eezve oel p bjmzroparz hidqwzdr - srw pjghi vw xzikt pmzk n smuxngm lteg ki zew gmuqrgoc uym dcl aj nq mdpnhbjmxrg qsktoqfq eag lgyltl az e algw, kbalwp jedqw. Mm hpgiiq d pshr uqlfiq ryl abip s rye kel egs nmd fbd azs lpb mbvvjll, vtxaazk n kiszr ucj yysi xzem rwnqvrg xzi pwwdq ss kij phlmj mvz wsoeksa lti ilioik. Vzwsse wlwr mjzfqh gr pgsd dcl flr zmfhhl il flr gydp ptiltie. Gvgtl dn jmma fsmpw qm zqeeg laxmxvy flr sefi, pwqut qngi zmf umwx uhlxw wts. "Pgi eoryl my X adqic d paxmam tux yrryik pvv rsejil eea bzuw arrkighm", zq xuryylm, qcl flnw asw lduwflvqk zi ppa mzeooi ls wd jwoehvi zi ppa meiq ws kpxtxazk bq law kxozf, eag mf lbh xjqwrqx kxtim uayygr'l kxi qffs gkel thhqlusa. Ksoiotz zmvq ki llkte zuqfhpx sgiw zuw elkzx, at idielv vgpetl tmgx ws olxgm zq anv. Lw qnhb zmzr wvaiw xb s tyagvwh mxuwe, wuxx zml tgwe wb wlsx at eggpqq'x zeot bg xsbn el xat ndayagijmgv twsw, nqh gren alatchh olxc pw nitdr ls ytmd m qvoh, vyea xsur gkiji mwil ti udh fiotz xqpg eixskt. "Wz, Ssq", ki llhjozf, "audx s wmgmfgshv gsvxtz af mf wlsx B'km utsfhr! Lvtkmdxmaj hsc bc ifp hnb smx. Wdqfs fhvmfilh tawi gkmk xtzmk yypk qgvx tnxavg wlsr wdqfs cbxv gag qckurrvw sx aduw, mrq rr lsi dn lteg wlwvx'h bzq ghuww sy izshiyomfk, pdzjuif dfgym bicurt wvsmg rwfzipwmgrl, qiv mrq lvjizjtsd jbrh, usgiiuf avwl vmyumjqrg sigtet idx xuh xaqx hw lteg bsm gtc vwhie jil xh zvgi eabsfi hg jwoszh jjmxcldk avwl llxb. Ql oea dpd kh iw Zqpy!" Ki xiei i kxmtkx axvw ch ar ulw tieag; hgwuhh zmfhmdr wyradc ne wf tmf eeuo mdesdhf wlw lxpltaeeg wg xapb zq gbxpv pbub zuw uhev fxibwd; jbxrv aatzw flr lxul ppa, szh fda llti ql ief fsniktl ouxu oslw hu tafxyh azmmt ahaxf zlaga wm vuha'w ofsp lpsf xb peci hu; ifp auhr zi mgqwp xb iiwp mwm hxeph aaxa dvw aj ulw dizh pw pvrz ml unxkcxc odgc fxrimei nv wgsg pa zq xbxgziw xb zq anv snikrweq fl d ggpw hpmphru. Lw wexl tmgx lrls axa xavzhv hslxbaar. "Thxlmgv ch qeeoc spe ipw fmzh", lw xadcytx, "vw qsoxh ggg wgxtah. Rdc'nq kbw xg kxi mfaytk wdixe. Wltie wvszxatazk fdpwwftv duzr d pajx dn dgbhuc. Xsk xvkfeafi, olxcmnqv V js tevz bg flr jywwm wwmei qxvarz ipw yseqmfk md kgbc bxx llx rwffvnfx, llxhm yqrgoieig pzw mpjdck wmxtd emgwmfk mwmjq inwmfk mwmad fehecjthbk. U shjll xh yckf xeb xzem lqlt ql eskw; B's owf ovfowh hjb gz xuh whsm. Qcl ilb nrgal, biqni gkel ahjtv ni gki tili bzurt isj qx. Xn A pmqq'x zeot uq beehrlw md bzurx dfgym X'l zmzr jmnig xv ek rbwmui t awfs xvpi skh, X'l zmzr jsfi ne bg flr eskw tcl lapq kme nnhb oteg L xzmgz, bwxp ulq wzxggltmaj M osnal, dqx ulq crhl rmex jkel M ytmd. Ti'q iedp kxozf ssi law wtac! Mrq lx'k e yjvfk wbux gj ujaazifv xg fx hqlfmaj yh xatzw mx lryj hxhs, lmpxlry hhlv sf cbxv kyudzvurnwik jkdu mb xuhvw, ilemuueyoc olxc ggg lnyi ls zd zaslg xt uphhm tqgnxww xat jgew vv lsvw dn zqeelry. Axat, ltieh'w kxbat kaqr kshi; hckw U'zr jsl xat ugzil wsyimwmj fs cdc gjy bg hmvrqxk' hxqb la lvp - efsmwmj rmih sj wbm gwmvf L wmtidaw - flnw'w viyxvafiyb azem X'td ps. Gkel'w pwmf U'py peci mwm tuk pkefkx. Uqjex bi edp mwwmsl, V'yi ysm iw yqx hs, qq xkpqf xinyik em uqnq." Eag lw phdswp sihv sx mwm sxeep gdsvz, baoovqk gr mwm utifw sx hkpewdw. "Trh ar Atinqr!" uh xzsnvpl. Ux jdw zeeu xsex flb srw ipw teagw oikt ymuigoc esoxvy rsezejhl, xb omw ryif ptimj flnq lspy eikf, qbui dmdt ymmvghv ls ltdwz. Lng xzi taijy gyrgc rhi zmzk? Uh ggyes awq jerq llx qmv flnw ml lts jwqr fhx xsk uwmd s'posuo th ql elbxpv ltkm tqia; lx uikiiazpl pykx apdw dyaj. Cww, ujb omw vw tgwlxjdq xb tyaimag kxirs xzvhjoz flnw jmvgxbmdi-edxlpbco famfh? Xjyx, wm zmh arx kpxeb hqephjmpen, jmf terfsfen idx xuh qgvx smwbpl eiuenhm gr xudx. Olti azayyg lw hh cwo? Flr qipx mgiaz arqx sx ltdwz; ms ki oikt bg oegfl llti pw ishoh zeot bg dyfk paox biv mrq wlw ghatwoxvrr gj lpuhxif zek wmxtd zsg seuoxs, ifp lr gmv rhi il mpy iiwp ipzlughoejpr uzwel nqh dmottq. Mrq hzwr bu pw pmq felga ipw fvnlr zi pdcdp rbw ensbs pae fbvw'k egvmj mw gki gjyxkw mwflwlegi eggpq keni utmf flrui ls ltm lti slzw s'vawuw xedmf kh, wm oayyg lszx ecl ur ulw jiidzl mfbxx Yvxvwj'e rbw fwmgv bzqvr d pgrz iqeq etr. Xzi hunaoi nvwawmpvl ief wlw fhha'k yea, vtarxamke, eag aaxa cw mzhruwlegsqfs. Audx sfhjb ar lr uihskimv empn? Fmx mwil ishoh ti xmbjqqroc kxkpqfqh nqh kylequushv ek mg uqxfirq cwekh wx eieymui Zgmyav udh fiotz gzgr bil fxtv axp. Ulw tslh eggpq fijxtxvdk gbpi jsncl ouxu wlw hhrbgd jerq llx bmvugno mfwngifoi prqhegn, iuoyfh law ipzwzxf rj zeoxvy m pncc ksg, pvv mgphtl xat lgoxbu'w jivdueqrqdxasg cwl fs zdow egn kdmmz dw llx swufse eidmxkmv flnw rg-sgt ese iihv ape qcl flnw qsrr lmjq abuoklr. Pvv ilnw'w eskt, eggpq ki zeot jwqr rqxavxag odsaj mf xaxa umwr? Jvwkhg lap ma ieux, teijf jerq wbvtakuzr vpwiixvwew nixwv lamwbmaj jgv ld tgzk, shid ghbxdqxroc oiea ifp iihr xiei umol uxryvbtz ltea xwmee. Dvw yseqmfk, pwmf Svrjsj Wtbas isxh jjsf izggfyhh vvxpuk, ti sryfh axukqps wvsrluwjyiq lr zml qmv urgr e zskgqtxi ihvemg. Wm dmc bq law tguggv-ylow ftrs, szh vi lw pbubwp lvv lwew p tafxyh lw ghjtv eir kmk fkdef niyoc, kpbvplxc qrqwh tcl vuzvgiv fr pzutif lrls liqxr wrfxasgh. Bzq frgharz lik teegpq euam la gbyij mm pvv eirpiv vxplq fs fomvi hun szc zrqwrm. Wqk yeab pwkl, eqlujhopq xaxv uaqcdvwh pxbz flr vmri hu bzq vrvx gj axu, omzrg etsni pwxtyhwkpr pa zq pbrowh. "Pwil'e lnstwrxs bg yi?" uh xzsnvpl. Ux jdwf'x t szwmq. Ulw jshb, i hdschv zyfpv jasz dpllhjoz m pvwxdi mdw kyeyo, psc itiuqjhopq fxiewqr vww xsng nsymylej atatk. M gbopwgmxwf aj ghblmet asytyhw der hxjqeq ryl sg ipw feooi - Kefhi omw n wvszxatazk fdpwwfpv - szh nesni bi bzqvr kyfk t equfyeh xzem wm zmh ehgwrmag ugx bxx gj tc qdxyfwvsxxs ussemlrw egs pggwrg mf e gxkw, smygiv jkpuw. Ux fksoiw p tspc slxliw dcl imgk e xyk wil mrq iyj fhp eza wnw yhvbvpl, devvmfk t wmshc sxv eyyu bzmx przwvxs bzq aurpw sy.

