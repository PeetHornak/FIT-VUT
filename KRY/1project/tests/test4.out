11;11;11;desetpismen
